storage/innobase/mtr/mtr0mtr.cc

```diff
@@ -1175,8 +1175,11 @@ ATTRIBUTE_COLD static void log_overwrite_warning(lsn_t age, lsn_t capacity)
     log_close_warn_time= t;
 
     sql_print_error("InnoDB: The age of the last checkpoint is " LSN_PF
-                    ", which exceeds the log capacity " LSN_PF ".",
+                    ", which exceeds the log capacity " LSN_PF "."
+                    " Shutdown is still in progress.",
                     age, capacity);
+    sql_print_information("InnoDB: Corruption fixed by log checkpoint at LSN=" LSN_PF ".",
+                          checkpoint);
```

Comment:
This message should actually be output by `log_t::write_checkpoint()` and only in the case the previous warning was issued. I was going to replace the `log_close_warned` with an `Atomic_relaxed<bool>` data member in `log_t`.